src/v/cluster/topics_frontend.cc

```diff
@@ -111,6 +130,24 @@ cluster::errc map_errc(std::error_code ec) {
     return errc::replication_error;
 }
 
+ss::future<topic_result> topics_frontend::do_update_topic_properties(
+  topic_properties_update update, model::timeout_clock::time_point timeout) {
+    update_topic_properties_cmd cmd(update.tp_ns, update.properties);
+    try {
+        auto ec = co_await replicate_and_wait(std::move(cmd), timeout);
+        co_return topic_result(std::move(update.tp_ns), map_errc(ec));
+    } catch (...) {
+        vlog(
+          clusterlog.warn,
+          "unable to update {} configuration properties - {}",
+          update.tp_ns,
+          std::current_exception());
+
+        co_return topic_result(
+          std::move(update.tp_ns), errc::replication_error);
+    }
```

Comment:
the function is missing a `coreturn errc:no_error` no?

Reply:
There is `co_return`, few lines above.